genesyscloud/resource_genesyscloud_routing_queue.go

```diff
@@ -1635,15 +1496,28 @@ func updateQueueUserRingNum(queueID string, userID string, ringNum int, api *pla
 }
 
 func getRoutingQueueMembers(queueID string, memberBy string, api *platformclientv2.RoutingApi) ([]platformclientv2.Queuemember, diag.Diagnostics) {
-	const maxPageSize = 100
-
 	var members []platformclientv2.Queuemember
+	const pageSize = 100
+
+	// Need to call this method to find the member count for a queue. GetRoutingQueueMembers does not return a `total` property for us to use.
+	queue, _, err := api.GetRoutingQueue(queueID)
+	queueMembers := *queue.MemberCount
+	if err != nil {
+		return nil, diag.Errorf("Can't find queue %s", queueID)
+	}
```

Comment:
I would move this error check up a line, before trying to access a value on the object `queue`